File: Problem Sets/pset1/mario.c
#include <cs50.h>
#include <stdio.h>

void printLeft(int, int);
void printRight(int);
void nextLine(void);
void printDoubleSpace(void);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (!(height > 0 && height < 9));

    for (int i = 1; i <= height; i++)
    {
        printLeft(height - i, i);
        printDoubleSpace();
        printRight(i);
        nextLine();
    }
}

void nextLine(void)
{
    printf("\n");
}
void printLeft(int numSpaces, int numHashes)
{
    for (int i = 1; i <= numSpaces; i++)
    {
        printf(" ");
    }
    for (int j = 1; j <= numHashes; j++)
    {
        printf("#");
    }
}

void printRight(int numHashes)
{
    for (int i = 1; i <= numHashes; i++)
    {
        printf("#");
    }
}

void printDoubleSpace(void)
{
    printf(" ");
    printf(" ");
}

File: Problem Sets/pset6/mario.py
from cs50 import get_int

def main():
    while True:
        height = get_int("Height: ")
        if height >= 0 and height <= 8:
            break

    for num_hashes in range(1, height + 1):

        num_spaces = height - num_hashes

        print(" " * num_spaces, end="")
        print("#" * num_hashes, end="")

        print("  ", end="")

        print("#" * num_hashes)


if __name__ == "__main__":
    main()